techtonica-projects/quiz-app/server/server.js

```diff
@@ -38,4 +38,6 @@ app.get('/api/trivia', async (req, res) => {
 });
 
 
-app.listen(PORT, () => console.log(`Server is runnning on port http://localhost:${PORT}`))
\ No newline at end of file
+app.listen(PORT, () => console.log(`Server is runnning on port http://localhost:${PORT}`))
+
+export default app;
```

Comment:
What were the other alternatives for fixing the issue you faced while configuring Supertest?
This export hack appears a bit of an overkill to me, maybe there's a simpler fix available?

Reply:
I added this line because I encountered an error that this syntax helped resolve, allowing the test to run successfully. I am continuing to troubleshoot and optimize the testing for the API.

```
    SyntaxError: The requested module '../server.js' does not provide an export named 'default'